src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -72,40 +72,41 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by all users, or empty list if no user has ever posted a
    *     message. List is sorted by time descending.
    */
-public List<Message> getAllMessages(){
+  public List<Message> getAllMessages(){
```

Comment:
Please don't change the indentation of code you haven't made changes to and try to follow the style guide. If you happen to be using IntelliJ or Eclipse for your development here are some instructions on how to install plugins to help keep the code conforming to the style guides.

http://www.practicesofmastery.com/post/eclipse-google-java-style-guide/
http://www.practicesofmastery.com/post/intellij-checkstyle-google-java-style-guide/